ONSdigital/eq-author
# Add previous answer to earliest and latest date

### What is the context of this PR?

Implements functionality which allows users to validate the earliest and latest date against a previous answer.

### How to review 
- Run tests
- Ensure feature works as per https://trello.com/c/UzNS13t8/451-validation-earliest-date-before-previous-answer-4 and https://trello.com/c/bnSemkJw/675-validation-latest-date-before-previous-answer-6
Dependencies

### Dependencies

- [x] https://github.com/ONSdigital/eq-author-api/pull/162
